Libraries/Components/Touchable/Touchable.js

```diff
@@ -758,7 +809,12 @@ const TouchableMixin = {
    * @param {Event} e Native event.
    * @sideeffects
    */
-  _performSideEffectsForTransition: function(curState, nextState, signal, e) {
+  _performSideEffectsForTransition: function(
+    curState: State,
+    nextState: State,
+    signal: Signal,
+    e: LayoutEvent,
+  ) {
```

Comment:
This event is forwarded to `_startHighlight` and `_endHighlight`. Shouldn't it therefore also be typed as a `PressEvent`?